build/windows/omsupply_desktop.suf

```diff
@@ -3045,7 +3077,28 @@ g_HandleSystemReboot();
 <Event>
 <Name>On Startup</Name>
 <Args/>
-<Script/>
+<Script>local sWindowTitle = "omSupply Desktop Setup";
+local sReadyToInstallSubTitle = SessionVar.Expand("You are now ready to install %ProductName% %ProductVer%.");
+
+hasOmsupplyDesktopInstalled = false;
+omSupplyDesktopPath = SessionVar.Expand("%AppFolder%\\open mSupply.exe");
+
+-- Check if the omSupply Desktop has been already installed on this machine
+if Registry.DoesKeyExist(HKEY_LOCAL_MACHINE, "Software\\omSupply Desktop") then

```

Comment:
I tried installing the test-21 installers and the registry keys were not created. are they working ok for you?